core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/job/report/DefaultReportJobDelegate.java

```diff
@@ -124,6 +127,8 @@ public void execute(final String reportKey) throws JobExecutionException {
         execution.setStatus(ReportExecStatus.STARTED);
         execution.setStart(new Date());
         execution.setReport(report);
+
+        execution.setExecutor(getAuthenticationPrincipal());
```

Comment:
Replace this with `org.apache.syncope.core.spring.security.AuthContextUtils#getUsername()`